src/main/java/kai/TaskList.java

```diff
@@ -0,0 +1,62 @@
+package kai;
+
+import java.util.ArrayList;
+
+import kai.tasks.Task;
+
+/**
+ * For managing the list of Task in an encapsulated manner
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Constructor of TaskList with the given tasks
+     *
+     * @param tasks the list of tasks in the class
+     */
+    public TaskList(ArrayList<Task> tasks) {
```

Comment:
Please note that it would be more convenient if there were a constructor that takes no argument

Reply:
Fair, I'll just this(new ArrayList<Task>()) this or something I suppose: fix it some other time, thanks.